### DISCUSSION TITLE
BentoML docker image size

### DISCUSSION THREAD

**Discussion by mat-berna** [bentoml/BentoML#1511] [Q&A] [answered]
Hi,

I create a simple ML service with BentoML using the sample described on BentoML quickstart (simple IrisClassifer with a random forest algorithm).
I want to containerize the BentoML bundle.
Everything works correctly but I noticed that the size of the generated image is more or less 500-600 MB. 
I think the image is bigger than expected.

Is it correct? 
Is there a slim version image?
If it can help I can also provide all the source code.

Thanks.

**Comment by parano** [comment, marked as answer]
@berna4 yes there's a slim image you can use, see documentation here: https://docs.bentoml.org/en/latest/concepts.html?highlight=slim#using-other-docker-base-images

Unlike the default debian based docker image, you will need to use your python version and bentoml version to find the right slim docker image to use, e.g.:
```python
@env(docker_base_image="bentoml/model-server:0.12.0-slim-py37")
@artifacts([SklearnModelArtifact('model')])
class ExamplePredictionService(BentoService):
  ...```

**Reply by mat-berna** [reply]
Hi @parano, thank you for the answer.
I compare created images using slim or "normal" bentoml base image as you can see below:


![BentoML_images](https://user-images.githubusercontent.com/9885388/111910654-92d82900-8a62-11eb-8da9-393251688993.png)


Thank you for all.
Best regards.

Matteo
